src/test/operators/join_test_runner.cpp

```diff
@@ -163,6 +173,7 @@ class JoinTestRunner : public BaseTestWithParam<JoinTestConfiguration> {
         {{{ColumnID{0}, ColumnID{0}}, PredicateCondition::LessThan}},
         {{{ColumnID{0}, ColumnID{0}}, PredicateCondition::GreaterThanEquals}},
         {{{ColumnID{0}, ColumnID{0}}, PredicateCondition::NotEquals}}};
+    const auto all_index_sides = std::vector{IndexSide::Left, IndexSide::Right};
```

Comment:
Just scrolling over the code. Do we also need to test `IndexSide::Left`? Could there be a case where to use both sides' indexes?

Reply:
>Do we also need to test IndexSide::Left?

Since both `IndexSide::Left` and `IndexSide::Right` are currently supported by the JoinIndex implementation, `IndexSide::Left` should be tested. If the index is on the left side, the swapping mechanism is triggered, which is implemented as part of the JoinIndex operator itself.

>Could there be a case where to use both sides' indexes?

I'm not sure if there could be a case where using both sides' indexes would be beneficial. However, we currently only support either left or right as an index side in the JoinIndex operator implementation.